src/main/kotlin/no/nav/familie/ef/sak/api/gui/BrevKontroller.kt

```diff
@@ -20,8 +20,15 @@ class BrevKontroller(private val brevService: BrevService,
 
     @PostMapping("/{behandlingId}")
     fun lagBrev(@PathVariable behandlingId: UUID, @RequestBody brevParams: BrevRequest): Ressurs<ByteArray> {
-        val respons = brevService.lagBrev()
+        val respons = brevService.lagBrev(behandlingId)
```

Comment:
Burde vi ha tilgangssjekk her? Hvis ikke finnes en liten risiko att man kan generere brev for gitt behandlingId som genererer data som man egentlige ikke har tilgang til :)

Reply:
Jepp, det burde vi 😊